yt_dlp/extractor/tiktok.py

```diff
@@ -133,7 +133,8 @@ def _call_api(self, ep, query, video_id, fatal=True,
                         else:
                             self.report_warning(str(e.cause or e.msg))
                             return
-                    self.report_warning('%s. Retrying... (attempt %s of %s)' % (str(e.cause or e.msg), count, len(self._APP_VERSIONS)))
+                    self.report_warning(
+                        '%s. Retrying... (attempt %s of %s)' % (str(e.cause or e.msg), count, len(self._APP_VERSIONS)))
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
                    self.report_warning('%s. Retrying... (attempt %s of %s)' % (str(e.cause or e.msg), count, len(self._APP_VERSIONS)))
```